File: BOJ/Java/단계별로_풀어보기/B_01_입출력과_사칙연산/BOJ_25083.java
package 단계별로_풀어보기.B_01_입출력과_사칙연산;

/**
 * BOJ 25083 : 새싹 
 * 
 * @author kdgyun
 * {@link https://st-lab.tistory.com}
 * {@link https://github.com/kdgyun}
 *
 */

public class BOJ_25083 {
	public static void main(String[] args) {
		String s = "         ,r'\"7\n" + 	// \", \n 이 제어문자다.
				   "r`-_   ,'  ,/\n" + 		// \n 이 제어문자다.
				   " \\. \". L_r'\n" + 		// \\, \", \n 이 제어문자다.
				   "   `~\\/\n" + 			// \\, \n 이 제어문자다.
				   "      |\n" + 			// \n 이 제어문자다.
				   "      |";

		System.out.println(s);
	}
}

File: BOJ/CPP/단계별로_풀어보기/01_입출력과_사칙연산/BOJ_25083.cpp
/*
    BOJ 25083 : 새싹

    author : kdgyun
    link https://st-lab.tistory.com
    link https://github.com/kdgyun
    Created by kdgyun
    Copyright © 2022 kdgyun. All rights reserved.
*/
#include <iostream>
#include <string>

using namespace std;

int main(int argc, char const *argv[]) {

	string s = "         ,r'\"7\n";   // \", \n 이 제어문자다.
	       s += "r`-_   ,'  ,/\n";    // \n 이 제어문자다.
	       s += " \\. \". L_r'\n";    // \\, \", \n 이 제어문자다.
	       s += "   `~\\/\n";         // \\, \n 이 제어문자다.
	       s += "      |\n";          // \n 이 제어문자다.
	       s += "      |";
	cout << s;
	return 0;
}
